docs/development/deploying-locally/index.md

```diff
@@ -1,3 +1,70 @@
 # Deploying locally
 
 REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+
+### Instructions
+
+* (1) [Fork the REANA main repository](https://github.com/reanahub/reana/fork) and clone it:
+
+```console
+$ git clone https://github.com/johndoe/reana.git
+$ cd reana
+$ mkvirtualenv reana && pip install . --upgrade
+```
+
+* (2) Fork all REANA cluster components repositories:
+
+```console
+(reana) $ reana-dev git-fork
+```
+
+* (3) Create a REANA development setup as follows:
+```console
+(reana) $ GITHUB_USER=johndoe make setup clone prefetch
```

Comment:
Take out `make clone` as an extra step between (2) and (3), so that people have a clear mind separation line that up to now we have been basically only cloning repositories, to have the source code available, and from now on we can boot minikube, prefetch images, and build the system.

This would lead to (1) fork and clone main reana repo to install reana-dev helper script (2) fork and clone all REANA repos to have full sources locally (3) start minikube and prefetch base docker images (4) launch REANA cluster component build (5) deploy (6) run example.

Reply:
One more thing: IIRC we need `reana-dev fetch -c reana` for the main repo because there will be no `upstream` for this repo, so some of the later scripts would fail?  Anyway, we can also update this bit by making `make clone` more intelligent... or just move to `hub` later on.